deploy/crds/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -32,6 +32,41 @@ spec:
         spec:
           description: ClusterLogForwarderSpec defines the desired state of ClusterLogForwarder
           properties:
+            inputs:
+              description: Inputs are named inputs of log messages
+              items:
+                description: InputSpec defines a source of log messages.
+                properties:
+                  application:
+                    description: Application specific specs
+                    properties:
+                      namespaceNames:
+                        description: "NamespaceNames referes to the namespace names.
+                          The input to the pipeline will be restricted to logs from
+                          containers belonging to these namespaces. \n If no namespaces
+                          are specified, all application container logs will be collected."
+                        items:
+                          type: string
+                        type: array
+                    required:
+                    - namespaceNames
+                    type: object
+                  name:
+                    description: Name used to refer to the input of a `pipeline`.
+                    type: string
+                  type:
+                    description: 'Type of input source. Must be one of :  - "Application"  -
+                      "Infrastructure"  - "Audit"'
```

Comment:
FYI: in #427 I changed to lowercase "application", "audit", "infrastructure". Since output types are also used as CR field names, they have to startLowerCase. It was too confusing mix that with StartUpperCase for inputs side-by-side in the pipeline spec so its all startLowerCase now.
If you merge before #427 then don't worry about it, I'll fix it when I merge #427.

Reply:
Updated